docs/tutorial.md

```diff
@@ -0,0 +1,48 @@
+Once you are in an environment with both cvmfs commands and pyhf supported, as well as the package installed, you are ready to run the translation
+
+### pyhf->Combine
+
+To go from a pyhf workspace to a datacard, you only need the workspace JSON file
+
+The translation can be run either via the CLI or with a python script. First, to run it in the CLI:
+
+```
+pyhf-to-combine $JSON_FILE_NAME
+```
+
+If you want to designate the names of the datacard and/or shapes files that are generated by the conversion, add the following flags respectively
+
+```
+--out-datacard $DATACARD_FILE_NAME
+--shape-file $SHAPES_FILE_NAME
+```
+
+To run the translation in a python script, run the following code:
+
+```
+from pyhf_combine_converter import pyhf_convert_to_datacard
+pyhf_convert_to_datacard.main($JSON_FILE)
```

Comment:
I am wondering now whether we should import `pyhf_convert_to_datacard.main` at the top level of the package and call it `pyhf_convert_to_datacard` to allow calling `pyhf_combine_converter.pyhf_convert_to_datacard()` directly, what do you think? (The downside of this is that the other functions inside `pyhf_convert_to_datacard` might not be easily accessible anymore then.)

Reply:
Yeah, I think this is a good idea.

Reply:
This was done in PR #28.